app/javascript/components/party/index.js

```diff
@@ -8,48 +8,46 @@ import Metrics from "./metrics";
 import Candidates from "./Candidates";
 
 export default class Party extends PureComponent {
+    constructor() {
+        super();
+
+        this.state = {
+            party: {},
+            candidates: []
+        }
+    }
+
+    componentDidMount() {
+      fetch("/parties/" + this.props.match.params.id + ".json")
```

Comment:
Why do we need the `.json` in the end?

Reply:
I think if we don't do that rails tries to render HTML instead of json. But I can double check

Reply:
Have you checked this? @luisiul359